test/hack/resource-management/clean-resources/main.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const expirationTTL = time.Hour * 12
+const karpenterMetricTableName = "sweeperCleanedResources"
```

Comment:
consider naming the variable to be the same as the variable value
```suggestion
const sweeperCleanedResourcesTableName = "sweeperCleanedResources"
```